cmd/ketch/builder.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+
+	"github.com/shipa-corp/ketch/cmd/ketch/configuration"
+)
+
+const builderCmdHelp = `
+Manage pack builders.
+
+A builder is an image that contains all the components needed to build your project into an image.
+There are already a number of builders available for use by all developers, as well as the option to build and use your own.
+
+You can learn more about builders at: https://buildpacks.io/docs/concepts/components/builder/
+`
+
+func newBuilderCmd(cfg config, out io.Writer) *cobra.Command {
```

Comment:
Thanks for pointing this out. None of the builder commands depend on any of the methods provided by the config interface. This also made me realize there was no reason to include the `KetchConfig` struct in the `Configuration` struct, since no other command outside of builder makes use of the `KetchConfig` struct.